pulpcore/cli/core/task.py

```diff
@@ -167,6 +170,33 @@ def cancel(
         task_ctx.cancel(task_ctx.pulp_href)
 
 
+@task.command()
+@href_option
+@uuid_option
+@click.option("--download/--no-download")
+@pass_task_context
+@pass_pulp_context
+def profile_artifact_urls(
+    pulp_ctx: PulpCLIContext,
+    task_ctx: PulpTaskContext,
+    /,
+    download: bool,
+) -> None:
+    """Prints download links for profile urls of the task."""
+    urls = task_ctx.profile_artifact_urls()
+    pulp_ctx.output_result(urls)
+    if download:
+        uuid_match = re.match(r".*/api/v3/tasks/(?P<uuid>.*)/", task_ctx.entity["pulp_href"])
+        assert uuid_match is not None
+        uuid = uuid_match.group("uuid")
+        with requests.session() as session:
+            for name, url in urls.items():
+                profile_artifact_path = Path(".") / f"task-{uuid}-{name}"
```

Comment:
I'm not against this structure necessarily, but wdyt about creating a directory named `task-{uuid}` and putting artifacts inside it?

Reply:
or even `{task_name} - {uuid}`.

Reply:
Creating a deeper directory structure is mainly a matter of taste...
Please have another look.